pkg/scrapper_test.go

```diff
@@ -18,11 +18,17 @@ type MockCloudWatchClient struct {
 	CloudWatchClient
 }
 
+// Int32 returns a pointer to the int32 value passed in
+func Int32(v int32) *int32 {
```

Comment:
I need to pass pointer to int32 as value to PeriodValue and this approach works, but please let me know if something else it better to use.